snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -294,4 +298,4 @@ private extension CGFloat {
 
 // Please don't remove the lines below
 // They are used to detect outdated configuration files
-// SnapshotHelperVersion [1.24]
+// SnapshotHelperVersion [1.25]
```

Comment:
This can be reverted as it's auto-increased as part of the release process 🙏

Reply:
Done